exampleCourse/questions/gallery/multipleChoice/gallery.md

```diff
@@ -14,7 +14,7 @@ Consequently, the correct answer is also fixed.
 
 ```
 
-This question uses the attribute `none-of-the-above="true"` in the `pl-multiple-choice` element. This attribute adds the option "None of the above" as an alternative among the other options defined by the `pl-answer` tags. The answer "None of the above" will be true (replacing the correct answer) with $50\%$ probability.
+This question uses the attribute `none-of-the-above="random"` in the `pl-multiple-choice` element. This attribute adds the option "None of the above" as an alternative among the other options defined by the `pl-answer` tags. The answer "None of the above" will be correct with probability proportional to the total number of correct choices (i.e., if there are N possible correct choices, "None of the above" will be correct with probability 1/N).
```

Comment:
Nit: the probability is `1/(N+1)`.

https://github.com/PrairieLearn/PrairieLearn/blob/e3712eeaaa6c6a872d9ae112358af8e8e80c6912/apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.py#L295-L301

That said, the provided code only has one correct answer, so the 50% would be correct in the provided example. Is it worth discussing implementation details about this probability in this example?